tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -412,6 +412,32 @@ def test_pantsd_multiple_parallel_runs(self):
       self.assert_success(creator_handle.join())
       self.assert_success(waiter_handle.join())
 
+  def test_pantsd_killed_run(self):
+    with self.pantsd_test_context() as (workdir, config, checker):
+      # Launch a run that will wait for a file to be created (but do not create that file).
+      file_to_make = os.path.join(workdir, 'some_magic_file')
+      waiter_handle = self.run_pants_with_workdir_without_waiting(
+        ['run', 'testprojects/src/python/coordinated_runs:waiter', '--', file_to_make],
+        workdir,
+        config,
+      )
+
+      # Wait for the python run to be running.
+      time.sleep(15)
```

Comment:
Could consider a retry strategy for this "chaos monkey" testing approach should this test ever become flaky.